docs/pages/access-graph/getting-started-on-prem.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Getting started with Teleport Access Graph (TAG)
+---
+
+<Notice type="note">
+Teleport Access Graph is provided by default to users of the Teleport Team plan.
+Visit the [signup page](https://goteleport.com/signup/) to begin your free&nbsp;trial.
+</Notice>
+
+## Prerequisites
+
+<Tabs>
+<TabItem
+  scope={["enterprise"]} label="Self-Hosted Enterprise">
+
+- A running Teleport Enterprise cluster v15.0.0 or later.
+- Docker version v(=docker.version=) or later.
+- A PostgresSQL database server v14 or later.
+  - Access Graph needs a dedicated [database](https://www.postgresql.org/docs/current/sql-createdatabase.html) to store its data.
+    The user that TAG connects to Postgres with needs to be an owner of this database, or have similar broad permissions:
+    at least the `CREATE TABLE` privilege on the `public` schema, and the `CREATE SCHEMA` privilege.
+- A TLS certificate for the Access Graph service
+- The node running the Access Graph service must be reachable from Teleport Auth and Proxy services.
+
+</TabItem>
+</Tabs>
+
+## Step 1/3. Set up the Teleport Access Graph service
+
+You'll need a copy of your Teleport cluster's host certificate authority (CA) on the machine that hosts the Access Graph service.
```

Comment:
Can we give a bit of insight why this is necessary? 
Refer that Auth and Proxy will use certificates signed by HostCA to dial to...